src/zoid/buttons/component.jsx

```diff
@@ -344,9 +346,7 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
             experiment: {
                 type:       'object',
                 queryParam: true,
-                value:      () => {
-                    return {};
-                }
+                value:      () => getVenmoExperiment(enableVenmoExperiment)
```

Comment:
The thinking here is there will be two ways to see the Venmo button:
1. Using the `enable-funding=venmo` query param
2. Being on the allowed list and getting bucketed into the treatment.